browser/main/SideNav/SideNav.styl

```diff
@@ -13,14 +13,20 @@
   height $topBar-height
   padding 0 15px
   font-size 12px
-  width 100%
-  text-align left
+  width 33%
+  text-align center
   &:hover
     color $ui-text-color
   &:active, &:active:hover
     color $ui-text-color
     background-color alpha($ui-button--active-backgroundColor, 20%)
 
+.active-button
+  background-color #c4c4c4
+
+.non-active-button
+  background-color white
```

Comment:
Manage color on some file is better. Like `browser/styles/vars.styl`.

Reply:
I think that's true.
Most of the styles used in this pull request at this stage will be changed later, but I will fix it.